exasol/ds/sandbox/runtime/ansible/roles/entrypoint/tasks/main.yml

```diff
@@ -0,0 +1,11 @@
+- name: Copy entry point script
+  ansible.builtin.copy:
+    src: "entrypoint.py"
+    dest: "/root/entrypoint.py"
```

Comment:
How does python know, where the entrypoint is located. and no hard-coded path in task. Move it to the variables. Similar to jupyterlab.notebook_folder

Reply:
OK, will do so.